contributor.md

```diff
@@ -1,6 +1,39 @@
-# No spam Prs that aim only to get a +1 count in Hacktoberfest are appreciated and will be marked `invalid`.
+# Contributing 
+
 <div style="text-align:center; width:100%">
 <img  src="https://github.com/Ishaan28malik/Get-Dev-Resources/blob/main/src/assets/Git-octo.gif" />
 </div>
 
-## Good PRs that solve any of the existing issues or upcoming issues and helps in improving the Repo quality and not quantity are welcome.
+## Cloning the Repository
+ - Fork the Repository, this is to take a copy of the whole repository into your own GitHub account.
+ - Next, navigate to your command line interface to clone the project on your local machine. 
+   You can do that with the `git clone https://github.com/yourUsername/theNameOfTheRepo.git` clone command.
+ 
+## Make a new branch other than master using
```

Comment:
Master is not being used for the main branch here, this should say `new branch other than main`